app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en" class="govuk-template ">
   <head>
-    <title>GOV.UK Rails Boilerplate</title>
+    <title>Register trainee teachers</title>
```

Comment:
Is the repo name wrong or is this or neither?

Reply:
no idea, i just borrowed this from the prototype as the out of the box one was irking me

Reply:
@misaka ^_^

Reply:
I think that'll probably get settled along the way... probably don't need to worry too much about it at the mo.

Reply:
I know... now i seen it also annoys me... :see_no_evil: